m4/tlog/play_conf_schema.m4

```diff
@@ -92,6 +92,12 @@ M4_PARAM(`/es', `query', `file-',
          `STRING is the ', `The ',
          `M4_LINES(`query string to send to Elasticsearch')')m4_dnl
 m4_dnl
+M4_PARAM(`/es', `verbose', `opts-',
```

Comment:
Please change the `opts-` here to `file-` to allow this option to be configured in the configuration file.